proxy/http2/HTTP2.cc

```diff
@@ -482,6 +482,11 @@ http2_convert_header_from_2_to_1_1(HTTPHdr *headers)
     headers->field_delete(HTTP2_VALUE_AUTHORITY, HTTP2_LEN_AUTHORITY);
     headers->field_delete(HTTP2_VALUE_PATH, HTTP2_LEN_PATH);
   } else {
+    // Set HTTP Version 1.1
+    int32_t version = HTTP_VERSION(1, 1);
```

Comment:
I'm ok with this, but curious, why can't this just be

```
http_hdr_version_set(headers->m_http, HTTP_VERSION(1,1);
```

Reply:
I guess it's because all other places do this way. Maybe @masaori335, the author of this change, knows.

Reply:
We can go with that. I just followed code 10 lines above from here.

Well, it looks like we should move this code to out side of this if-else block. Now, it's common code regardless `http_hdr_type`

Reply:
Let's leave it as is. I don't want to spend extra time for this trivial changes. We'll need to touch this code anyway in the future, because H3 module depends on H2 module due to this function.